active_model_serializers.gemspec

```diff
@@ -40,7 +40,8 @@ Gem::Specification.new do |spec|
   # 'i18n,
   # 'tzinfo'
   spec.add_development_dependency 'minitest', ['~> 5.0', '< 5.11']
-  # 'thread_safe'
+
+  spec.add_dependency 'thread_safe','~> 0.3', '>= 0.3.4'
```

Comment:
can we be explicit here: `add_development_dependency` or `add_runtime_dependency`

Reply:
I assumed when running tests against Rails 7.1 it will be needed as well as in production.